spec/requests/api/v1/moves_controller_filter_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Api::V1::MovesController, with_client_authentication: true do
+  let(:headers) { { 'CONTENT_TYPE': content_type }.merge(auth_headers) }
+  let(:content_type) { ApiController::CONTENT_TYPE }
+  let(:response_json) { JSON.parse(response.body) }
+
+  describe 'GET /moves' do
+    context 'when filtering' do
+      let!(:supplier) { create :supplier }
+      let!(:location) { create :location, :with_moves, suppliers: [supplier] }
+      let!(:filtered_out_moves) { create_list :move, 10 }
+      let(:filter_params) { { filter: { supplier_id: supplier.id } } }
+
+      before do
+        next if RSpec.current_example.metadata[:skip_before]
```

Comment:
Do we need this? I can't see it being used in this file?

Reply:
Gone